project/plugins.sbt

```diff
@@ -2,4 +2,6 @@ addSbtPlugin("io.spray" % "sbt-revolver" % "0.8.0")
 
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.0-M7")
+addSbtPlugin("se.marcuslonnberg" % "sbt-docker" % "1.4.1")
```

Comment:
Just wanted to double check that this is an industry standard repo, not just "some guy's project"?